src/main/groovy/org/jetbrains/intellij/tasks/IntelliJInstrumentCodeTask.groovy

```diff
@@ -0,0 +1,108 @@
+package org.jetbrains.intellij.tasks
+
+import org.apache.tools.ant.BuildException
+import org.gradle.api.DefaultTask
+import org.gradle.api.Nullable
+import org.gradle.api.file.ConfigurableFileCollection
+import org.gradle.api.file.FileCollection
+import org.gradle.api.file.SourceDirectorySet
+import org.gradle.api.tasks.*
+import org.jetbrains.annotations.NotNull
+import org.jetbrains.intellij.IntelliJPlugin
+import org.jetbrains.intellij.dependency.IdeaDependency
+
+class IntelliJInstrumentCodeTask extends DefaultTask {
```

Comment:
you right

> I'm not sure if I can do much to make it display the file as diff.

You can rename file and change its content in two different commits